src/explore-education-statistics-frontend/.env.dev02

```diff
@@ -3,3 +3,5 @@ DATA_API_BASE_URL=https://s101d01-as-eesdev02-data.azurewebsites.net
 FUNCTION_API_BASE_URL=https://s101d01-fa-eesdev02-notify.azurewebsites.net
 GA_TRACKING_ID=UA-139424173-3
 GA_TRACKING=true
+HOTJAR_TRACKING=true
```

Comment:
Ideally we only want this enabled on live so as to not skew any stats with our local development / testing habbits. But happy to merge this in to allow us to test it provided we revert this before marking as complete